althea_kernel_interface/src/exit_client_tunnel.rs

```diff
@@ -127,6 +127,25 @@ impl dyn KernelInterface {
             }
         }
 
+        // If wg_exit does not have a link local addr, set one up
+        if self.get_link_device_ip_v6("wg_exit").is_err() {
+            //copy another wg interface and use the same linklocal addr
+            match self.get_link_device_ip_v6("wg0") {
```

Comment:
Look at how we do this in open_tunnel.rs in the open_interface function, you can re-use that method of generating an fe80 address here rather than depending on wg0 existing to copy it.

Reply:
Fixed